src/state-indicators.service.js

```diff
@@ -191,11 +198,16 @@ class StateIndicatorsService {
       return !isZoneStockCount(stockCount)
     }
 
+    const isNationalStockCount = (stockCount) => {
+      return (stockCount.location && stockCount.location.national)
+    }
+
     const decorateStockCounts = (nonZoneStockCounts, zoneStockCounts, promiseResults) => {
       lgas = promiseResults.lgas
       states = promiseResults.states
       zones = promiseResults.zones || [] // not available for the state dashboard
       products = promiseResults.products
+      national = promiseResults.national || []
```

Comment:
`national = promiseResults.national || {}` would be better because `promiseResults.national` (when not undefined) will be an object